### PR TITLE
🌐 Add Korean translation for `/docs/ko/docs/environment-variables.md`

### PR DESCRIPTION
**Pull request by Tolerblanc**
Add Korean Translation for `/docs/ko/docs/environment-variables.md`

### CODE REVIEW

**Comment by sodaMelon** [comment]
@alejsdev hi, could you re-check with github-actions? 
new page is not being created in this PR

**Comment by tiangolo** [comment]
Thanks @sodaMelon! The problem with the missing previews was handled here https://github.com/fastapi/fastapi/pull/12527

**Comment by sodaMelon** [comment]
> Thanks @sodaMelon! The problem with the missing previews was handled here #12527

oh hi @tiangolo, thanks!!! what a awesome speed lmao
have a nice day

File: docs/ko/docs/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+# 환경 변수
+
+/// tip | "팁"
+
+만약 "환경 변수"가 무엇이고, 어떻게 사용하는지 알고 계시다면, 이 챕터를 스킵하셔도 좋습니다.
+
+///
+
+환경 변수는 파이썬 코드 **바깥**, **운영 체제** 단에 존재하는 변수입니다. 파이썬 코드나 다른 프로그램에서 읽을 수 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:9]:
```suggestion
환경 변수는 파이썬 코드의 **바깥**인, **운영 체제** 에 존재하는 변수입니다. 파이썬 코드나 다른 프로그램에서 읽을 수 있습니다.
```

File: docs/ko/docs/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+환경 변수는 애플리케이션 **설정**을 처리할 때나, 파이썬의 **설치** 과정의 일부로 유용할 수 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:11]:
```suggestion
환경 변수는 애플리케이션 **설정**을 처리하거나, 파이썬의 **설치** 과정의 일부로 유용합니다.
```

File: docs/ko/docs/environment-variables.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+## 환경 변수를 만들고 사용하기
+
+파이썬을 사용할 필요 없이, **셸 (터미널)** 에서 환경변수를 **생성** 하고 사용할 수 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:15]:
```suggestion
파이썬 없이도, **셸 (터미널)** 에서 환경변수를 **생성** 하고 사용할 수 있습니다.
```

File: docs/ko/docs/environment-variables.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+Hello Wade Wilson
+```
+
+</div>
+
+////
+
+## 파이썬에서 환경 변수 읽기
+
+파이썬 **바깥** 인 터미널에서 환경 변수를 생성할 수 있고(물론, 다른 방법도 좋습니다.), **파이썬에서 그것들을 읽을 수 있습니다.**
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:55]:
```suggestion
파이썬 **바깥** 인 터미널에서(다른 수단도 가능합니다.) 환경 변수를 생성할 수 있고, **파이썬에서 그것들을 읽을 수 있습니다.**
```

File: docs/ko/docs/environment-variables.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+
+/// tip | "팁"
+
+<a href="https://docs.python.org/3.8/library/os.html#os.getenv" class="external-link" target="_blank">`os.getenv()`</a> 의 두 번째 인자는 반환할 기본값입니다.
+
+여기서는 `"World"`를 넣었기에 기본값으로써 사용됩니다. 넣지 않으면 `None` 이 기본값으로 사용됩니다.
+
+///
+
+그러면 그 파이썬 프로그램을 호출할 수 있습니다:
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:74]:
```suggestion
이렇게 그 파이썬 프로그램을 실행할 수 있습니다:
```

File: docs/ko/docs/environment-variables.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+
+</div>
+
+////
+
+환경변수는 코드 바깥에서 설정될 수 있지만, 코드에서 읽을 수 있고, 나머지 파일과 함께 저장(`git`에 커밋)할 필요가 없으므로, 구성이나 **설정** 에 사용하는 것이 일반적입니다.
+
+**특정 프로그램 호출**에 대해서만 사용할 수 있는 환경 변수를 만들 수도 있습니다. 해당 프로그램에서만 사용할 수 있고, 해당 프로그램이 실행되는 동안만 사용할 수 있습니다.
+
+그렇게 하려면 프로그램 바로 앞, 같은 줄에 환경 변수를 만듭니다:
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:134]:
```suggestion
그렇게 하려면 프로그램 바로 앞, 같은 줄에 환경 변수를 만들어야 합니다:
```

File: docs/ko/docs/environment-variables.md [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+
+이 환경변수들은 오직 **텍스트 문자열**로만 처리할 수 있습니다. 텍스트 문자열은 Python 외부에 있으며 다른 프로그램 및 나머지 시스템(Linux, Windows, macOS 등 다른 운영 체제)과 호환되어야 합니다.
+
+즉, Python에서 환경 변수로부터 읽은 **모든 값**은 **`str`**이 되고, 다른 타입으로의 변환이나 검증은 코드에서 수행해야 합니다.
+
+**애플리케이션 설정**을 처리하기 위한 환경 변수 사용에 대한 자세한 내용은 [고급 사용자 가이드 - 설정 및 환경 변수](./advanced/settings.md){.internal-link target=\_blank} 에서 확인할 수 있습니다.
+
+## `PATH` 환경 변수
+
+**`PATH`** 라고 불리는, 운영체제(Linux, Windows, macOS 등)에서 실행할 프로그램을 찾기위해 사용되는 **특별한** 환경변수가 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:170]:
```suggestion
**`PATH`**라고 불리는, **특별한** 환경변수가 있습니다.  운영체제(Linux, Windows, macOS 등)에서 실행할 프로그램을 찾기위해 사용됩니다.
```
modified more clear without ``,``

File: docs/ko/docs/environment-variables.md [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+
+### 파이썬 설치와 `PATH` 업데이트
+
+파이썬을 설치할 때, 아마 `PATH` 환경 변수를 업데이트 할 것이냐고 물어봤을 겁니다.
+
+//// tab | Linux, macOS
+
+Python을 설치하고 그것이 `/opt/custompython/bin` 디렉토리에 있다고 가정해 보겠습니다.
+
+`PATH` 환경 변수를 업데이트하도록 예라고 하면 설치 관리자가 `/opt/custompython/bin`을 `PATH` 환경 변수에 추가합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:220]:
```suggestion
`PATH` 환경 변수를 업데이트하도록 "예"라고 하면 설치 관리자가 `/opt/custompython/bin`을 `PATH` 환경 변수에 추가합니다.
```

File: docs/ko/docs/environment-variables.md [hunk 9]
```diff
@@ -0,0 +1,298 @@
...
+
+다음과 같이 보일 수 있습니다:
+
+```plaintext
+/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/custompython/bin
+```
+
+이렇게 하면 터미널에 `python`을 입력하면 시스템이 `/opt/custompython/bin`(마지막 디렉토리)에서 Python 프로그램을 찾아 사용합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:228]:
```suggestion
이렇게 하면 터미널에 `python`을 입력할 때, 시스템이 `/opt/custompython/bin`(마지막 디렉토리)에서 Python 프로그램을 찾아 사용합니다.
```

File: docs/ko/docs/environment-variables.md [hunk 10]
```diff
@@ -0,0 +1,298 @@
...
+
+////
+
+//// tab | Windows
+
+Python을 설치하고 그것이 `C:\opt\custompython\bin` 디렉토리에 있다고 가정해 보겠습니다.
+
+`PATH` 환경 변수를 업데이트하도록 예라고 하면 설치 관리자가 `C:\opt\custompython\bin`을 `PATH` 환경 변수에 추가합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:236]:
```suggestion
`PATH` 환경 변수를 업데이트하도록 "예"라고 하면 설치 관리자가 `C:\opt\custompython\bin`을 `PATH` 환경 변수에 추가합니다.
```

File: docs/ko/docs/environment-variables.md [hunk 11]
```diff
@@ -0,0 +1,298 @@
...
+
+```plaintext
+C:\Program Files\Python312\Scripts;C:\Program Files\Python312;C:\Windows\System32;C:\opt\custompython\bin
+```
+
+이렇게 하면 터미널에 `python`을 입력하면 시스템이 `C:\opt\custompython\bin`(마지막 디렉토리)에서 Python 프로그램을 찾아 사용합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:242]:
```suggestion
이렇게 하면 터미널에 `python`을 입력할 때, 시스템이 `C:\opt\custompython\bin`(마지막 디렉토리)에서 Python 프로그램을 찾아 사용합니다.
```

File: docs/ko/docs/environment-variables.md [hunk 12]
```diff
@@ -0,0 +1,298 @@
...
+$ python
+```
+
+</div>
+
+//// tab | Linux, macOS
+
+시스템은 `/opt/custompython/bin`에서 `python` 프로그램을 **찾아** 실행합니다.
+
+대략 다음과 같이 입력하는 것과 같습니다:
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:260]:
```suggestion
다음과 같이 입력하는 것과 거의 같습니다:
```

File: docs/ko/docs/environment-variables.md [hunk 13]
```diff
@@ -0,0 +1,298 @@
...
+
+</div>
+
+////
+
+//// tab | Windows
+
+시스템은 `C:\opt\custompython\bin\python`에서 `python` 프로그램을 **찾아** 실행합니다.
+
+대략 다음과 같이 입력하는 것과 같습니다:
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:276]:
```suggestion
다음과 같이 입력하는 것과 거의 같습니다:
```

File: docs/ko/docs/environment-variables.md [hunk 14]
```diff
@@ -0,0 +1,298 @@
...
+
+</div>
+
+////
+
+이 정보는 [가상 환경](virtual-environments.md){.internal-link target=\_blank} 에 대해 알아볼 때 유용할 것입니다.
+
+## 결론
+
+이를 통해 **환경 변수**가 무엇이고 Python에서 어떻게 사용하는지에 대한 기본적인 이해가 생겼을 것입니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:292]:
```suggestion
이 문서를 읽고 **환경 변수**가 무엇이고 Python에서 어떻게 사용하는지 기본적으로 이해하셨을 겁니다.
```

File: docs/ko/docs/environment-variables.md [hunk 15]
```diff
@@ -0,0 +1,298 @@
...
+
+또한 <a href="https://ko.wikipedia.org/wiki/Environment_variable" class="external-link" target="_blank">환경 변수에 대한 위키피디아(한국어)</a>에서 이에 대해 자세히 알아볼 수 있습니다.
+
+많은 경우 환경 변수가 어떻게 유용하고 적용 가능한지 바로 알 수 없습니다. 하지만 개발할 때 다양한 시나리오에서 계속 나타나므로 이에 대해 아는 것이 좋습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/environment-variables.md:296]:
```suggestion
많은 경우에서, 환경 변수가 어떻게 유용하고 적용 가능한지 바로 명확하게 알 수는 없습니다. 하지만 개발할 때 다양한 시나리오에서 계속 나타나므로 이에 대해 아는 것이 좋습니다.
```

**Review by sodaMelon — COMMENTED** [review]
hi.
To make sure your sentences read naturally and clearly, i left some suggestions.

**Comment by Tolerblanc** [comment]
@sodaMelon Thank you for your detailed review! I have revised everything you pointed out.

**Review by sodaMelon — APPROVED** [review]
LGTM 👍

**Comment by sodaMelon** [comment]
hi @hard-coders @JaeHyuckSa 
could you review this PR?

File: docs/ko/docs/environment-variables.md [hunk 16]
```diff
@@ -0,0 +1,298 @@
...
+
+<a href="https://12factor.net/config" class="external-link" target="_blank">The Twelve-Factor App: Config</a> 에서 좀 더 자세히 알아볼 수 있습니다.
+
+///
+
+## 타입과 검증(Validation)
+
+이 환경변수들은 오직 **텍스트 문자열**로만 처리할 수 있습니다. 텍스트 문자열은 Python 외부에 있으며 다른 프로그램 및 나머지 시스템(Linux, Windows, macOS 등 다른 운영 체제)과 호환되어야 합니다.
+
+즉, Python에서 환경 변수로부터 읽은 **모든 값**은 **`str`**이 되고, 다른 타입으로의 변환이나 검증은 코드에서 수행해야 합니다.
```
Comment (hard-coders) [inline docs/ko/docs/environment-variables.md:164]: Python -> 파이썬

File: docs/ko/docs/environment-variables.md [hunk 17]
```diff
@@ -0,0 +1,298 @@
...
+
+찾으면 **사용합니다**. 그렇지 않으면 **다른 디렉토리**에서 계속 찾습니다.
+
+### 파이썬 설치와 `PATH` 업데이트
+
+파이썬을 설치할 때, 아마 `PATH` 환경 변수를 업데이트 할 것이냐고 물어봤을 겁니다.
+
+//// tab | Linux, macOS
+
+Python을 설치하고 그것이 `/opt/custompython/bin` 디렉토리에 있다고 가정해 보겠습니다.
```
Comment (hard-coders) [inline docs/ko/docs/environment-variables.md:218]: Python -> 파이썬

File: docs/ko/docs/environment-variables.md [hunk 18]
```diff
@@ -0,0 +1,298 @@
...
+
+Python을 설치하고 그것이 `C:\opt\custompython\bin` 디렉토리에 있다고 가정해 보겠습니다.
+
+`PATH` 환경 변수를 업데이트하도록 "예"라고 하면 설치 관리자가 `C:\opt\custompython\bin`을 `PATH` 환경 변수에 추가합니다.
+
+```plaintext
+C:\Program Files\Python312\Scripts;C:\Program Files\Python312;C:\Windows\System32;C:\opt\custompython\bin
+```
+
+이렇게 하면 터미널에 `python`을 입력할 때, 시스템이 `C:\opt\custompython\bin`(마지막 디렉토리)에서 Python 프로그램을 찾아 사용합니다.
```
Comment (hard-coders) [inline docs/ko/docs/environment-variables.md:242]: 마지막 미번역 "파이썬"... Python -> 파이썬

File: docs/ko/docs/environment-variables.md [hunk 19]
```diff
@@ -0,0 +1,298 @@
...
+
+////
+
+이 정보는 [가상 환경](virtual-environments.md){.internal-link target=\_blank} 에 대해 알아볼 때 유용할 것입니다.
+
+## 결론
+
+이 문서를 읽고 **환경 변수**가 무엇이고 Python에서 어떻게 사용하는지 기본적으로 이해하셨을 겁니다.
+
+또한 <a href="https://ko.wikipedia.org/wiki/Environment_variable" class="external-link" target="_blank">환경 변수에 대한 위키피디아(한국어)</a>에서 이에 대해 자세히 알아볼 수 있습니다.
```
Comment (hard-coders) [inline docs/ko/docs/environment-variables.md:294]:
```suggestion
또한 <a href="https://ko.wikipedia.org/wiki/환경_변수" class="external-link" target="_blank">환경 변수에 대한 위키피디아(한국어)</a>에서 이에 대해 자세히 알아볼 수 있습니다.
```

한국어 위키로 링크하는 것은 좋은 아이디어라고 생각합니다.
괄호로 한국어 위키임을 명시적으로 나타낸 것도 매우 좋습니다. 👍🏻 
다만 링크가 잘못되어있습니다.

**Comment by Tolerblanc** [comment]
@hard-coders  Thank you for your review! I corrected the ambiguous parts and the parts written Python to `파이썬`.

**Review by hard-coders — APPROVED** [review]
LGTM 🚀

**Comment by alejsdev** [comment]
Thanks everyone for the help! @Tolerblanc @sodaMelon @hard-coders :raised_hands:

**Pull request merged**